meta/main.yml

```
@@ -2,17 +2,41 @@
 galaxy_info:
   author: Shane Frasier
   description: Install cisagov/cyhy-feeds.
-  company: CISA NCATS
+  company: CISA Cyber Assessments
+  galaxy_tags:
+    - cyhy
+    - feeds
   license: CC0
-  min_ansible_version: 2.0
+  # With the release of version 2.10, Ansible finally correctly
+  # identifies Kali Linux as being the Kali distribution of the Debian
+  # OS family.  This simplifies a lot of things for roles that support
+  # Kali Linux, so it makes sense to force the installation of Ansible
+  # 2.10 or newer.
+  min_ansible_version: 2.10
+  namespace: cisagov
   platforms:
+    # - name: Amazon
+    #   versions:
+    #     - 2
     - name: Debian
       versions:
         - stretch
-  galaxy_tags:
-    - cyhy
-    - feeds
+    #     - buster
```

Don't we want buster included here?
```suggestion
        - buster
```

---

This PR just skeletonizes the repo.  #9 actually uses the new `pip` and `python` Ansible roles to expand the platforms.

---

Ah, sorry- I missed that distinction, thanks!